types/react-reconciler/test/ReactSymbols.ts

```diff
@@ -0,0 +1 @@
+export const REACT_OPAQUE_ID_TYPE: number | symbol = 0xeae0;
```

Comment:
What is this testing?

Reply:
Just paste from facebook reconciler test host config, I have move this into test file self and delete this file